lighthouse-core/test/scripts/i18n/collect-strings-test.js

```diff
@@ -155,44 +186,137 @@ describe('Compute Description', () => {
       /** 
        * @description Tagged description for Hello World.
        */
-      message: 'Hello World {variable}',
+      exampleString: 'Hello World {variable}',
     };`;
+    const liveUIStrings = evalJustUIStrings(justUIStrings);
+    const res = collect.parseUIStrings(justUIStrings, liveUIStrings);
 
-    const ast = esprima.parse(justUIStrings, {comment: true, range: true});
+    expect(res).toEqual({
+      exampleString: {
+        message: 'Hello World {variable}',
+        description: 'Tagged description for Hello World.',
+        examples: {},
+      },
+    });
+  });
+
+  it('throws when @example is blank', () => {
+    const justUIStrings =
+    `const UIStrings = {
+        /** 
+         * @description Some description.
+         * @example
+         */
+        exampleString: 'Hello World',
+    };`;
+    const liveUIStrings = evalJustUIStrings(justUIStrings);
+
+    expect(() => collect.parseUIStrings(justUIStrings, liveUIStrings))
+      .toThrow(/Incorrectly formatted @example/);
+  });
 
-    const stmt = ast.body[0];
-    const prop = stmt.declarations[0].init.properties[0];
-    const res = collect.computeDescription(ast, prop, 'Hello World {variable}', 0);
-    expect(res.description).toBe('Tagged description for Hello World.');
-    expect(res.examples).toEqual({});
+  it('throws when @example is missing a placeholder name', () => {
+    const justUIStrings =
+    `const UIStrings = {
+        /** 
+         * @description Some description.
+         * @example {missingPlaceholdername}
+         */
+        exampleString: 'Hello World',
+    };`;
+    const liveUIStrings = evalJustUIStrings(justUIStrings);
+
+    expect(() => collect.parseUIStrings(justUIStrings, liveUIStrings))
+      .toThrow(/Incorrectly formatted @example/);
```

Comment:
Is this error descriptive enough inside a stack trace?
![image](https://user-images.githubusercontent.com/6392995/62245777-25601c00-b397-11e9-8d66-f056638a384c.png)

I think it needs more context around it, what message was bad, or what `@example` text it was iterating over.  This would help immediately know what is wrong.

Reply:
> I think it needs more context around it, what message was bad, or what @example text it was iterating over. This would help immediately know what is wrong.

good point. added example string to error message